routes/index.js

```diff
@@ -1,43 +1,33 @@
 const express = require('express');
-const jose = require('node-jose');
 const IDPartner = require('@idpartner/node-oidc-client');
 
-let idPartnerClient = {};
-
-!async function () {
-  // This will generate a keypair however for production you will want to keep the private key secret
-  const keyStore = jose.JWK.createKeyStore();
-  await keyStore.generate('RSA', 2048, { alg: 'RSA-OAEP', enc: 'A256CBC-HS512', use: 'enc' });
-  await keyStore.generate('RSA', 2048, { alg: 'PS256', use: 'sig' });
-  
-  idPartnerClient = new IDPartner({
-    jwks: keyStore.toJSON(true),
-    client_id: 'CHANGE_ME_CLIENT_ID',
-    callback: 'https://myapplication.ngrok.io/auth/callback',
-});
-}();
-
 const router = express.Router();
 
-router.get('/', async (req, res, next) => {
-  res.render('index', { title: 'RP Example' });
+const idPartner = new IDPartner({
+  client_id: 'CHANGE_ME_CLIENT_ID',
+  client_secret: 'CHANGE_ME_CLIENT_SECRET',
+  callback: 'http://localhost:8000/auth/callback',
```

Comment:
should this be ngrok?

Reply:
not necessary, it works without and we make it simple, i include it in the further steps if u want to customize other things

Reply:
Discussed offline - the reason for not using ngrok is that this URL is consumed via redirect. It's not consumed via direct request (as it used to due to JWKS)